app/src/main/java/com/group80/uoftinder/Login.java

```diff
@@ -0,0 +1,87 @@
+public class Login extends AppCompatActivity {
+    private FirebaseAuth mAuth;
+
+    EditText loginEmail;
+    EditText loginPassword;
+    Button enterLogin;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.loginview);
+
+        mAuth = FirebaseAuth.getInstance(); // initialize the Firebase Auth
+
+        // UserAccountController
+        loginEmail = findViewById(R.id.loginEmail);
+        loginPassword = findViewById(R.id.password);
+
+        LoginPresenter loginPresenter = new LoginPresenterFormatter(Login.this, HelloWorld.class);
+        LoginInput loginInteractor = new LoginInteractor(loginPresenter);
+        LoginController loginController = new LoginController(loginInteractor);
+
+
+        enterLogin = findViewById(R.id.EnterLogin);
+        enterLogin.setOnClickListener(view -> loginController.loginUser(mAuth, loginEmail, loginPassword));
+
+    }
+
+    // LoginInteractor
+//    public void loginUser(FirebaseAuth mAuth, EditText loginEmail, EditText loginPassword) {
+//
+//        String email = loginEmail.getText().toString().trim();
+//        String password = loginPassword.getText().toString().trim();
+//
+//        /* To log-in, use the following
+//         * mAuth.signInWithEmailAndPassword("demo@email.com", "password");
+//         * To get user uid, after you have signed in, do the following
+//         * mAuth.getCurrentUser().getUid();
+//         */
+//        if (TextUtils.isEmpty(email)) { // no email input, user == null
+//            loginEmail.setError("Email is required!");
+//            loginEmail.requestFocus();
+//        }
+//        else if (TextUtils.isEmpty(password)){ // no password input, user == null
+//            loginPassword.setError("Password is required!");
+//            loginPassword.requestFocus();
+//        }
+//        else { // user == null, signing in with email and password
+//            mAuth.signInWithEmailAndPassword(email, password)
+//                    .addOnCompleteListener(this, new OnCompleteListener<AuthResult>() {
+//                        @Override
+//                        public void onComplete(@NonNull Task<AuthResult> task) {
+//                            // LoginInteractor checks
+//
+//                            // Login Presenter?
+//                            if (task.isSuccessful()) {
+//                                Toast.makeText(Login.this, "Login Successful!", Toast.LENGTH_SHORT).show();
+//                                startActivity(new Intent(Login.this, HelloWorld.class));
+//
+//                            } else {
+//                                // If sign in fails, display a message to the user.
+//                                Toast.makeText(Login.this,
+//                                        "Login Failed :(",
+//                                        Toast.LENGTH_SHORT).show();
+//                            }
```

Comment:
Just a quick question here - want to double check that this Firebase Auth logic is commented out for now because we need to wait for the `getUserById` method from Realtime Database Interactor?

Reply:
This was the old code prior to implementing clean architecture, I just kept it there in case we need it if something doesn't work out.